### ISSUE TITLE
[Spotlight] Alice "Through the looking glass"

### ISSUE THREAD

**Issue by devangm** [TAMULib/exhibits#104]
**URL:** https://spotlight.library.tamu.edu/spotlight/alice-in-wonderland/catalog/6ea42b8d915520c992f96ee1ed5c61af
**WCAG 2.1 AA compliant? :** No
**Violation:** 1.4.3 Contrast Minimum (AA)
**Issues:**  Contrast Error
**Notes:**  Digital collections nav bar selected item text doesn't have sufficient contrast

**Comment by qtamu** [comment]
See this nav bar for reference https://library.tamu.edu/collections/maps/
